.github/workflows/release-canary.yml

```
@@ -0,0 +1,33 @@
+name: release-canary
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  publish:
+    if: ${{ github.event.label.name == 'build' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+      # Ensure that the README is published with the package
+      - run: rm -f packages/cli/README.md && cp README.md packages/cli
+      - run: npm ci --workspace packages/cli
+      - run: npm version $(npm run version --workspace packages/cli --silent)-canary-${{github.event.pull_request.number}}.$(git rev-parse --short HEAD) --workspace packages/cli
+      - run: npm publish --workspace packages/cli --tag canary-${{github.event.pull_request.number}}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          header: NPM package canary
+          hide_and_recreate: true
+          hide_classify: "OUTDATED"
+          message: You can install the canary version using `npm install @checkly/cli@canary-${{github.event.pull_request.number}}`
```

```suggestion
          message: |
            Successfully published `@checkly/cli@canary-${{github.event.pull_request.number}}` 🎉
            
            ```
            npm install @checkly/cli@canary-${{github.event.pull_request.number}}
            ```
```

Little bit "cleaner" imo, and the triple backticks `npm install ..` command at the bottom of the message will include the code block copy btn, so people can copy it out of the comment with 1 simple click